definitions/ext-coredns/dashboard.json

```diff
@@ -22,13 +23,97 @@
           }
         },
         {
-          "title": "Sum of DNS panics",
+          "title": "Num of CoreDNS instances reporting UP",
           "layout": {
             "column": 3,
             "row": 1,
+            "width": 2,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "dataFormatters": [],
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "FROM Metric SELECT uniqueCount(instance) where instrumentation.provider = 'prometheus' AND k8s_app LIKE '%dns' AND metricName = 'up'  "
```

Comment:
@RagalahariP is `k8s_app` a standard attribute for the CoreDNS metrics?  If not, it shouldn't be referenced in the default quickstart dashboards.

Reply:
@bpschmitt  , Thank you for review.  As CoreDNS exposes prometheus metrics on port 9I53 (by default),  `k8s_app`  information is not an emitted metric from coreDNS . Instead, it is derived from CoreDNS endpoint labels to determine the number of CoreDNS application instances running.
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/71118025/193791983-32bf4f8f-d442-4f2f-897c-2238e00700ae.png">